cmd/core-common-config-bootstrapper/res/configuration.yaml

```diff
@@ -2,8 +2,8 @@ all-services:
   Writable:
     InsecureSecrets:
       DB:
-        path: "redisdb"
-        Secrets:
+        SecretName: "redisdb"
+        SecretData:
           username: ""
           password: ""
```

Comment:
Need to temporary support old and new until Bootstrap changes propagated to all services.
New will be ignored for those services and old will be ignored for updated services.
```
      DB:
        path: "redisdb"
        Secrets:
          username: ""
          password: ""
        SecretName: "redisdb"
        SecretData:
          username: ""
          password: ""
```